e2e_tests/data/yolov8/from/test_input_with_bboxes_and_polys.json

```diff
@@ -61,9 +61,7 @@
             {
               "x": 158.18,
               "y": 242.01
-            }
-          ],
-          [
+            },
```

Comment:
YOLOv8 doesn't support complex polygons, so adjust the convert test input to be simple polygons